README.md

```diff
@@ -1,61 +1,31 @@
 ![Travis report](https://travis-ci.org/ninja-ide/ninja-ide.svg?branch=master "Travis-C.I. Testing report")
 
-# Ninja-ide Is Not Just Another IDE.
-**Ninja-IDE** is a cross-platform integrated development environment (IDE) that allows developers to create applications for any purpose making the task of writing software easier and more enjoyable. It's also a secret ninja agency but this doesn't matter right now.
-![Ninja-IDE logo](http://ninjaide.webfactional.com/static/common/img/ninja-big.png)
-
+# Unofficial Repository:
```

Comment:
I was willing to check this change until I saw this line.
I'm not part of the core devs, so take my opinion as what it is, just the one of someone who's been in opensource projects for a while.
Of course you can have your own 'unofficial' fork, but if you want your changes merged into the core you don't change this kind of things or at least keep it in a separate branch so they don't go in your pull request.
Just my two cents.